package.json

```diff
@@ -19,10 +19,12 @@
   ],
   "files": [
     "dist/",
+    "index.d.ts",
```

Comment:
I've gotten issues reported with problems of the typings file being in a different directory than the module itself. Can we put it in `src` and make sure it's copied to `dist/` and referenced there?

... Unless those issues are totally unfounded and there's something I'm missing?